source/lib/rocprof-sys/library/rocprofiler-sdk.cpp

```diff
@@ -553,7 +1076,7 @@ tool_tracing_callback(rocprofiler_callback_tracing_record_t record,
             case ROCPROFILER_CALLBACK_TRACING_MEMORY_COPY:
 #if(ROCPROFILER_VERSION >= 600)
             case ROCPROFILER_CALLBACK_TRACING_OMPT:
-            case ROCPROFILER_CALLBACK_TRACING_MEMORY_ALLOCATION:
+            // case ROCPROFILER_CALLBACK_TRACING_MEMORY_ALLOCATION:
```

Comment:
Should we bring this back?
```suggestion
            case ROCPROFILER_CALLBACK_TRACING_MEMORY_ALLOCATION:
```